lib/both.js

```diff
@@ -73,10 +73,8 @@ DocHead = {
   removeDocHeadAddedTags() {
     if (Meteor.isClient) {
       const elements = document.querySelectorAll('[dochead="1"]');
-      for (let key in elements) {
-        if (elements.hasOwnProperty(key)) {
-          elements[key].parentNode.removeChild(elements[key]);
```

Comment:
I assume this is the best working solution.
May be we need to double check into that error message.